server/setup.py

```diff
@@ -28,5 +28,16 @@
     install_requires=[
         'blinker', 'celery >=3.1.0, <3.2.0', DJANGO_REQUIRES, 'httplib2', 'iniparse',
         'isodate>=0.5.0', 'm2crypto', 'mongoengine>=0.10.0', 'oauth2>=1.5.211', 'pymongo>=3.0.0',
-        'semantic_version>=2.2.0', 'setuptools']
+        'semantic_version>=2.2.0', 'setuptools'],
+    data_files=[
+        (
+            '/usr/share/pulp/templates/', 
+            [
+                'usr/share/pulp/templates/404.html',
```

Comment:
The way I read the Django 1.4 documentation, it didn't seem like it. It certainly wasn't serving anything visually useful before I did this, but maybe that's because we were returning HttpResponseNotFound instead of raising Http404.